src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -79,6 +79,7 @@ class java_bytecode_parsert:public parsert
     if(index==0 || index>=constant_pool.size())
     {
       error() << "invalid constant pool index (" << index << ")" << eom;
+      error() << "constant pool size: " << constant_pool.size() << eom;
```

Comment:
I think it makes still sense to output the accessed `index`, too

Reply:
This is still done on the line above.